composer.json

```diff
@@ -61,18 +66,20 @@
     }
   ],
   "require-dev": {
+    "php": "^7.4 || ^8.0",
     "alleyinteractive/alley-coding-standards": "^0.3.0",
+    "alleyinteractive/pest-plugin-wordpress": "^0.4.0",
     "alleyinteractive/searchpress": "^0.4.1",
     "automattic/vip-go-mu-plugins-built": "dev-master",
     "composer/installers": "^1.0",
-    "phpunit/phpunit": "7.5.*",
     "wpackagist-plugin/elasticpress": "^4.2",
     "yoast/phpunit-polyfills": "^1.0"
   },
   "scripts": {
     "phpcbf": "phpcbf .",
     "phpcs": "phpcs .",
-    "phpunit": "phpunit",
+    "phpunit": "@test",
```

Comment:
Don't bother aliasing this, just run `composer test`. That will need to be updated in the GitHub actions for tests as well, if it isn't already.